packages/flutter/test/rendering/rendering_tester.dart

```diff
@@ -47,6 +47,7 @@ TestRenderingFlutterBinding get renderer => _renderer;
 
 void layout(RenderBox box, { BoxConstraints constraints, EnginePhase phase: EnginePhase.layout }) {
   assert(box != null); // if you want to just repump the last box, call pumpFrame().
+  assert(box.parent == null); // we stick the box in another, so you can't reuse it easily, sorry
```

Comment:
seems like these comment and the previous one should either both have a . or neither.